anker/tests/tests/amm.rs

```diff
@@ -0,0 +1,39 @@
+use anker::token::MicroUst;
+use lido::token::{Lamports, StLamports};
+use solana_program_test::tokio;
+use solana_sdk::signer::Signer;
+use testlib::anker_context::Context;
+
+#[tokio::test]
+async fn test_successful_token_swap() {
+    let mut context = Context::new_with_initialized_token_pool().await;
+    let (st_sol_keypair, st_sol_token) = context
+        .solido_context
+        .deposit(Lamports(10_000_000_000))
+        .await;
+
+    let ust_address = context
+        .solido_context
+        .create_spl_token_account(
```

Comment:
We might add a `create_ust_token_account` shorthand to the `AnkerContext` to make this test a bit more high-level to read.